workspaces/mama/temp-test/package.json

```diff
@@ -0,0 +1 @@
+{"name":"test","version":"1.0.0"}
```

Comment:
? If you need a fixture you need to add a proper fixture folder in /test (also think to add the end of line).